apps/api/src/controller/guild/index.ts

```diff
@@ -0,0 +1,236 @@
+import express from 'express';
+import krebit from '@krebitdao/reputation-passport';
+import LitJsSdk from '@lit-protocol/sdk-nodejs';
+import { connect, guildXyz } from '../../utils';
+
+const {
+  SERVER_EXPIRES_YEARS,
+  SERVER_TRUST,
+  SERVER_STAKE,
+  SERVER_PRICE,
+  SERVER_CERAMIC_URL
+} = process.env;
+
+export const mergeArray = (arr: any[]): any[] => {
```

Comment:
would be great to have this method in the sdk to use it in the frontned and backend of the app